### DISCUSSION TITLE
Use WIFI and python (without using ROS2) to send commands to iRobot Create 3

### DISCUSSION THREAD

**Discussion by valdivj** [iRobotEducation/create3_docs#409] [Questions]
### How are you connecting to your Create 3?

Bluetooth (Python SDK)

### Computer(s) Model(s) and Operating System(s)

jetson agx orin

### Which version of ROS 2 is installed on your computer?

Galactic

### Which firmware version is installed on your robot?

g.4.5

### Which RMW is your robot running?

CycloneDDS

### Are there multiple Create 3 robots connected to your network?

No

### Is multicast enabled?

No

### What is the Adapter Board's USB/BLE Toggle currently switched to?

Bluetooth (default)

### Describe your question.

Are there any examples that use just WIFI and python (Not using ROS2)
to send data to and from I robot create 3

**Comment by kwujciak** [comment]
If you just want to use Python, you can try the [Python Playground](https://python.irobot.com/) or [Python SDK](https://github.com/iRobotEducation/irobot-edu-python-sdk/tree/main). [This page](https://edu.irobot.com/learning-library/create-3-web-playground-for-python) has directions and a "getting started" PDF. The [Python Playground](https://python.irobot.com/) has a bunch of pre-loaded python files that you can run on your Create just using bluetooth (and a stable wifi connection). You won't need a compute board like the Jetson, just your computer with a bluetooth compatible browser and the Create.
